### PR TITLE
add getAllEventsSegmentsForApp.js file

### PR DESCRIPTION
**Pull request by LaNinja06**

### CODE REVIEW

File: bin/scripts/export-data/getAllEventsSegmentsForApp.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * Command: node getallEventsSegmentsForApp.js
+ */
+
+var plugins = require("../../../plugins/pluginManager.js");
+
+plugins.dbConnection("countly").then(function(db) {
+    db.collection("events").find().toArray(function(err, events) {
+        db.collection("apps").find().toArray(function(err, names) {
+
+            if (err) {
```
Comment (ar2rsawseen) [inline bin/scripts/export-data/getAllEventsSegmentsForApp.js:14]: Here we need to check for both errors from events and app, so the names or arguments in those callbacks should be different and in the if you would check both with || operator

File: bin/scripts/export-data/getAllEventsSegmentsForApp.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * Server: countly
+ * Path: bin/scripts/export-data
+ * Command: node getallEventsSegmentsForApp.js
+ */
+
+var plugins = require("../../../plugins/pluginManager.js");
+
+plugins.dbConnection("countly").then(function(db) {
+    db.collection("events").find().toArray(function(err, events) {
+        db.collection("apps").find().toArray(function(err, names) {
```
Comment (ar2rsawseen) [inline bin/scripts/export-data/getAllEventsSegmentsForApp.js:12]: Here from app documents we only use name property, so let's provide it in projection, so we read from disk only that property and not whole document and not consume too much RAM to store whole app object there

**Review by ar2rsawseen — CHANGES_REQUESTED** [review]

File: bin/scripts/export-data/getAllEventsSegmentsForApp.js (hunk 1 above)
Comment (LaNinja06) [reply bin/scripts/export-data/getAllEventsSegmentsForApp.js]: I modified the code to reflect these changes! :)

File: bin/scripts/export-data/getAllEventsSegmentsForApp.js (hunk 2 above)
Comment (LaNinja06) [reply bin/scripts/export-data/getAllEventsSegmentsForApp.js]: I've modified the code to specify only the name property! :)

**Review by ar2rsawseen — APPROVED** [review]

**Pull request merged**